packages/main/tests/robot/test_robocorp_workitems.robot

```diff
@@ -40,12 +41,25 @@ Read input and write output
     Save work item
 
     Create output work item
-    Set work item variables    use=Another    mail=another@company.com
+    Set work item variables    user=Another    mail=another@company.com
     Save work item
     File should exist    ${temp_out}
 
     [Teardown]  Remove file     ${temp_out}
 
+Explicit state set
+    ${payload} =     Get Work Item Payload
+    Log     ${payload}
+    ${input} =      Get Current Work Item
+
+    Create Output Work Item
+    Set Work Item Variables    user=Another2    mail=another2@company.com
+    Save work item
+
+    Set Current Work Item   ${input}
+    Set Work Item State     SUCCESS
```

Comment:
@osrjv Are you cool with this idiom on setting the state like this for an **input** work item once we created previously multiple other children **output** work items? Look above at the helper `${input} =      Get Current Work Item` to get an idea on what I'm talking about.